Eat_and_Greet/src/style.tsx

```diff
@@ -1,25 +1,37 @@
-import { StyleSheet } from 'react-native';
+import { StyleSheet } from "react-native";
 
 const style = StyleSheet.create({
-    CornerButtonOutline: {
-        alignItems: 'center',
-        justifyContent: 'center',
-        position: 'relative',
-        backgroundColor: '#8EA7E9',
-        borderRadius: 10,
-        borderWidth: 2
-    },
-    ButtonText: {
-        fontFamily: 'Courier New',
-        fontSize:36,
-        fontWeight:'bold'
-    },
-    ScreenContainer: {
-        flex: 1,
-        backgroundColor: '#7286D3',
-        alignItems: 'center',
-        justifyContent: 'center',
-      }
-    });
+  CornerButtonOutline: {
+    alignItems: "center",
+    justifyContent: "center",
+    position: "relative",
+    backgroundColor: "#8EA7E9",
+    borderRadius: 10,
+    borderWidth: 2,
+  },
+  ButtonText: {
+    fontFamily: "Courier New",
+    fontSize: 36,
+    fontWeight: "bold",
+  },
+  ScreenContainer: {
+    flex: 1,
+    backgroundColor: "#7286D3",
+    alignItems: "center",
+    justifyContent: "flex-start",
+  },
+});
 
-export default style;
\ No newline at end of file
+export const navHeaderStyles = StyleSheet.create({
+  headerStyle: {
```

Comment:
nit: headerStyle -> headerColor

Reply:
headerStyle is predetermined naming with native navigation

Reply:
Sounds good!